uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/manager/leave-requests-show-actions.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences-manager-leave-requests');
+var capturePath = './backstop_data/screenshots/reference/';
+
+// precondition: need to have the login of manager and all status types having atleast one leave request
+module.exports = function (casper) {
```

Comment:
I think this test case is not necessary as we discussed earlier. Just a screenshot for the whole page of Manager Leave Report with open filters section would be enough. And we dont need to open any actions here.
cc @AkA84